test/fixtures/traverse/case-1/result.js

```diff
@@ -0,0 +1,40 @@
+// THESE ARE FOR ESM IMPORTS
+import assert from 'assert';
+import traverse from 'neotraverse/legacy';
```

Comment:
@PuruVJ can we add a test case for the `require` case as well?

Reply:
Done